examples/Digital filtering/FIRuncFilter_runtime_comparison.py

```diff
@@ -0,0 +1,55 @@
+"""Perform a speed comparison for uncertainty.propagate_filter.FIRuncFilter
+ with different value-types of the covariance of theta
+"""
+
+import numpy as np
+import time
+import pandas as pd
+
+from PyDynamic.uncertainty.propagate_filter import FIRuncFilter
+
+def evaluate(signal, filter, case="full/null/none"):
+    # different covariance matrices
+    if case == "full":
+        U_theta = 1e-2 * np.eye(filter_length)
+    elif case == "null":
```

Comment:
Should we call this case "zero" to avoid confusion with [null pointers](https://www.freecodecamp.org/news/a-quick-and-thorough-guide-to-null-what-it-is-and-how-you-should-use-it-d170cea62840/)? I know we don't have these in Python, but still...